cabal-testsuite/PackageTests/NewSdist/DeterministicTrivial/deterministic.test.hs

```diff
@@ -24,6 +24,5 @@ main = cabalTest $ do
     known <- liftIO (BS.readFile knownSdist)
     unknown <- liftIO (BS.readFile mySdist)
 
-    skipIf "#8356" True -- bogus, just to indicate that the test is skipped
```

Comment:
I approve this with a heavy heart because of how much pain "DeterministicTrivial" has brought us due to its flakiness in the past (what a name for such a record!..). I truly hope that it's fixed otherwise we're doooomed...